### PR TITLE
retrieval app refactoring no. 1

### PR DESCRIPTION
**Pull request by RobinBially**
retrieval app restructuring

### CODE REVIEW

File: backend/open_webui/apps/retrieval/search/api.py
Comment (tjbck) [inline backend/open_webui/apps/retrieval/search/api.py:1]: Should be in `main.py`

**Comment by tjbck** [comment]
The rest seems reasonable!

File: backend/open_webui/apps/retrieval/process/api.py
Comment (tjbck) [inline backend/open_webui/apps/retrieval/process/api.py:1]: All api endpoints should be in main.py

**Comment by tjbck** [comment]
Will cherry-pick later, Thanks!

**Comment by RobinBially** [comment]
I have removed the routers but now the pull request is gone :/

**Pull request closed without merging**